client/src/helpers/index.js

```diff
@@ -187,3 +194,40 @@ export const computeDistances = (userLatitude, userLongitude, stakeholders) => {
 
   return filteredStakeholders;
 };
+
+// input  {day: 'SUN', time: '01:00AM'}
+export function getNextDateForDay(customizedOpenTimeFilter, targetTimezone) {
+  const dayInput = customizedOpenTimeFilter.day;
```

Comment:
Nit: I'd recommend refactoring this to 
`const { day: dayInput, time: timeInput } = customizedOpenTimeFilter;`

the code is cleaner and more DRY 😄